app/logic/utils.py

```diff
@@ -42,3 +43,20 @@ def getStartofCurrentAcademicYear(currentTerm):
         fallTerm = Term.select().where(Term.year==currentTerm.year-1, Term.description == f"Fall {currentTerm.year-1}").get()
         return fallTerm
     return currentTerm
+
+def format24HourTime(unformattedTime):
```

Comment:
I really like how you changed the variable name to be clear what data it contains